<launch>


  <arg name="server" default="192.168.50.26"/>

  <node pkg="vrpn_client_ros" type="vrpn_client_node" name="vrpn_client_node">
    <rosparam subst_value="true">
      server: $(arg server)
      port: 3883

      update_frequency: 100.0
      frame_id: world

      # Use the VRPN server's time, or the client's ROS time.
      use_server_time: false
      broadcast_tf: true

      # Must either specify refresh frequency > 0.0, or a list of trackers to create
      refresh_tracker_frequency: 1.0
      #trackers:
      #- FirstTracker
      #- SecondTracker
    </rosparam>
  </node>

  <node name="go1_pose_filter" pkg="medos-demo" type="pose-filter.py" output="screen">
    <param name="input_topic" value="/vrpn_client_node/go1/pose" />
    <param name="output_topic" value="/cairo_go1/pose" />
  </node>

  <node name="bebop_pose_filter" pkg="medos-demo" type="pose-filter.py" output="screen">
    <param name="input_topic" value="/vrpn_client_node/bebop/pose" />
    <param name="output_topic" value="/cairo_bebop/pose" />
  </node>
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find medos-demo)/config/mocap-display.rviz"/>
</launch>